src/components/backend-ai-settings-view.ts

```diff
@@ -21,12 +21,35 @@ import '@vaadin/vaadin-grid/vaadin-grid';
 import '@vaadin/vaadin-grid/vaadin-grid-sort-column';
 
 import '@material/mwc-switch/mwc-switch';
-import '@material/mwc-select';
+import {Select} from '@material/mwc-select';
+import {TextField} from '@material/mwc-textfield';
 import '@material/mwc-list/mwc-list-item';
 
 import './lablup-activity-panel';
 import {default as PainKiller} from './backend-ai-painkiller';
 
+type BackendAIDialog = HTMLElementTagNameMap['backend-ai-dialog'];
+
+interface OptionAndId {
+  option: string;
+  id: string;
+}
+
```

Comment:
I think the name of interface is vague. Instead of using simple "Options", how about using `SettingOptions`?

Reply:
How about `SettingOption`? Because this type has single option-id pair.